content/release-10-15-0/announcements-10-15-0.md

```diff
@@ -23,9 +23,9 @@ How will users be affected? If the user is hosting an application with a key and
 
 ##### Improved security for OAI-Secure users
 
-As announced with [release 10.14](/release-10-14-0/announcements-10-14-0), as of release 10.16+, when an administrator changes permissions inside a role, users who have this specific role assigned and use OAI-Secure will be logged out.
+As announced with [release 10.14](/release-10-14-0/announcements-10-14-0), as of release 10.16+, when an administrator changes permissions inside a role, the change impacts users who have the specific role assigned, and use OAI-Secure.
 
-The purpose of this measure is to enhance security and to make sure that all OAI-Secure users have access to the relevant information in Cumulocity IoT, without compromising secure access to the information. We recommend that administrators announce this change and notify about the change, because the logged-in users will immediately be logged out, and all their changes will get lost. Therefore this action requires scheduling.
+The purpose of this measure is to enhance security and to make sure that all OAI-Secure users have access to the relevant information in Cumulocity IoT, without compromising secure access to the information. We recommend that administrators announce this change and notify about the change, because the logged-in users will be affected and if they try to perform an operation that is affected by the changed role, they will receive access denied, and will not be able to perform the operation. Therefore this action is recommended to be followed up with notification to the users, and scheduling.
```

Comment:
```suggestion
The purpose of this measure is to enhance security and to make sure that all OAI-Secure users have access to the relevant information in Cumulocity IoT, without compromising secure access to the information. 

We recommend that administrators announce this change, because logged-in users will be impacted. If they try to perform an operation that is affected by the changed role, they will receive an "access denied" error and will not be able to perform the operation. Therefore this action is recommended to be followed up with notification to the users and scheduling.
```